tests/test_color_utils.py

```
@@ -0,0 +1,92 @@
+
+    def test_already_base_color(self):
+        assert normalize_color_name("navy") == ["navy"]
+
+    def test_pine_green_black(self):
+        result = normalize_color_name("Pine Green/Black")
+        assert sorted(result) == ["black", "green"]
+
+
+class TestHexToHsl:
+    def test_pure_red(self):
+        h, s, l = hex_to_hsl("#FF0000")
```

Ruff (0.15.5)

[error] Ambiguous variable name: `l`

(E741)